test/statistic_test.cc

```diff
@@ -403,7 +403,7 @@ TYPED_TEST(SinkableStatisticTest, EmptySinkableStatistic) {
   EXPECT_EQ(Envoy::Stats::Histogram::Unit::Unspecified, stat.unit());
   EXPECT_FALSE(stat.used());
   EXPECT_EQ("", stat.name());
-  EXPECT_DEATH(stat.tagExtractedName(), ".*");
+  EXPECT_EQ("", stat.tagExtractedName());
```

Comment:
Are these changes intended to be in this PR? If yes, can we not the reason behind them in the PR description as it isn't obvious.

Reply:
Yes, it is intended. Added to PR description